.github/workflows/bot-pr-updated.yml

```diff
@@ -1,65 +1,18 @@
+name: OnSyncPR
+
 on:
   pull_request_target:
     types: [opened,synchronize]
-name: OnSyncPR
+
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
   cancel-in-progress: true
+
 jobs:
   check-format:
-    runs-on: ubuntu-22.04
-    if: github.event.pull_request.author_association == 'COLLABORATOR' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'OWNER'
-    steps:
-      - name: Checkout the latest code (shallow clone)
-        uses: actions/checkout@v3
-        with:
-          ref: develop
-      - name: Check for formatting changes
-        env:
-          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
-        run: cp .github/check-format.sh /tmp && /tmp/check-format.sh
-      - name: Upload code formatting patch
-        if: failure()
-        uses: actions/upload-artifact@v3
-        with:
-          name: patch
-          path: format.patch
-  abidiff:
```

Comment:
abidiff is deleted accidentally

Reply:
yes, I will revert that, when I prepare this branch for merging